### ISSUE TITLE
crash when using new version

### ISSUE THREAD

**Issue by VaporeonScripts** [AppliedEnergistics/Applied-Energistics-2#8269]
### Describe the bug

https://mclo.gs/0cpgZJc

### How to reproduce the bug

no

### Expected behavior

no crash

### Additional details

arch

### Which minecraft version are you using?

1.21

### On which mod loaders does it happen?

NeoForge

### Crash log

https://mclo.gs/0cpgZJc

**Comment by VaporeonScripts** [comment]
when i update the mod, it crashes :/

**Comment by 62832** [comment]
Yes, that's because MEGA Cells hasn't been updated yet to work with the latest release of AE2. Be patient.

**Comment by VaporeonScripts** [comment]
alright, ty for clarifing
